prow/update-jobs/Dockerfile

```diff
@@ -0,0 +1,35 @@
+# --- Args --- 
```

Comment:
Change `ARG` to `ENV` for any values being used in the image such as `GOPATH` which might be used in the entrypoint script.

Reply:
Done. Align environment to goversion build argument